### PR TITLE
fix!: remove non read-only ens methods

### LINKED ISSUE #6096: ENS package has too much functionality
#### Expected behavior
Web3.js only needs read methods for ens. Users should use ens SDK or UI for write methods

#### Steps to reproduce the behavior

1. [First step]
2. [Second step]
3. [and so on...]

### PR DESCRIPTION
**Pull request by mpetrunic**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).

Fixes #6096

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

**Review by avkos — CHANGES_REQUESTED** [review]
what issue is it fixed? if we do not have it will be better to create one.
And also, It reduced coverage. so it will be great to add some unit tests.

File: packages/web3-eth-ens/src/registry.ts [hunk 1]
```diff
@@ -48,23 +47,6 @@ export class Registry {
 		}
 	}
 
-	public setOwner(
```
Comment (jdevcs) [inline packages/web3-eth-ens/src/registry.ts:51]: As these are breaking changes so if you have time this should be mentioned in same PR in **migration guide** ? or we can add in some other PR?

**Review by jdevcs — APPROVED** [review]
some of tests in CI are not passing.

**Comment by mpetrunic** [comment]
> some of tests in CI are not passing.

Can somebody else spin them locally? For me they are passing locally :/

**Review by luu-alex — APPROVED** [review]

**Review by avkos — APPROVED** [review]

**Review by luu-alex — APPROVED** [review]

**Pull request merged**